docs/source/software/r.rst

```diff
@@ -28,3 +29,71 @@ You can then install R packages that you need. For example, to install a package
    if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
    BiocManager::install("rtracklayer")
+
+**When using Rstudio in the OOD App:**
+
+1. Launch an RStudio instance on the OOD.
```

Comment:
It might be worth mentioning any flavor would work here. How it reads people might think it is only the the RStudio options and not TidyVerse or Geospatial.